src/components/client/core/Menu/Menu.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import { Button, Menu } from "components/client";
+import { Text } from "components/universal";
+import { HStack } from "generated/panda/jsx";
+
+import type { Meta, StoryObj } from "@storybook/react";
+import type { MenuItem, MenuItemGroup } from "components/client";
+
+type Story = StoryObj<typeof meta>;
+
+const SUBMENU_GROUP_ITEMS: MenuItem[] = [
+  { id: "item-7", child: "Item 7" },
+  { id: "item-8", child: "Item 8" },
+  { id: "item-9", child: "Item 9" },
+];
+
+const SUBMENU_GROUP: MenuItemGroup = {
+  id: "group-3",
+  label: "Group 3",
+  items: SUBMENU_GROUP_ITEMS,
+};
+
+const GROUP_ONE_ITEMS: MenuItem[] = [
+  {
+    id: "item-1",
+    child: (
+      <HStack justify="space-between" flex={1}>
+        Item 1
+        <Text color="fg.subtle" textStyle="xs">
+          ⇧⌘P
```

Comment:
Is this a connected keyboard shortcut, or just a demonstration of custom text in a menu item? In either case, can we change this content to be OS-agnostic?

<img width="144" alt="2023-08-16_20-27" src="https://github.com/animareflection/ui/assets/20056195/5322fdfd-c2aa-430d-8372-0808441479b6">

Reply:
macOS or you are fired.... lol jk jk it is just a demonstration of how to represent more than just `string` types in the item child components. Can definitely adjust!

Reply:
How many times have I been fired now?

Reply:
Apparently not enough......